webbpsf/tests/test_opds.py

```diff
@@ -343,7 +343,8 @@ def test_apply_field_dependence_model():
     dependence. Thus there are several calls to manually set only the nominal field dep to True
 
     """
-    rms = lambda array, mask: np.sqrt((array[mask] ** 2).mean())
+    def rms(array, mask):
+        return np.sqrt((array[mask] ** 2).mean())
```

Comment:
This exact function is already defined in `utils.py` so we could just import it from there rather than defining a copy here.